### PR TITLE
removed ceph, from helm_chart

### PR DESCRIPTION
**Pull request by luciosis**
**Description of your changes:**

Checklist:

PR Author:

* [ ] I have bumped the chart version
* [ ] If necessary, I have documented my changes
* [ ] If writing a guide I have followed it my self
* [ ] I have described my changes in the "description of your changes" field
* [ ] I have tested (and noted how) this change on a running cluster by deploying a new cluster with the PR branch
* [ ] I have tested (and noted how) this change on a running cluster by migrating from the master branch to the PR branch and pasted the results in the comments
* [ ] If possible, automatic tests have been created for the change
* [ ] My commit messages describe why, not how
* [ ] I have linked my PR to a task and dragged it into the peer review column

PR Reviewer:

* [ ] I have understood why this PR exists
* [ ] I have given thought to, whether this is the best solution for the problem
* [ ] If my suggestions further expand the scope of the task, it should be suggested in the PR, but created as a separate task
* [ ] After reviewing I have dragged the task into the implement feedback column
* [ ] I encourage developers to simplify code or add code comments instead of just explaining the complexity to me

### CODE REVIEW

**Review by RolfTherkildsenAtWork — APPROVED** [review]

**Review by jaxels10 — CHANGES_REQUESTED** [review]
We need to write in the changelog what the change is so our tenants can upgrade with a positive mind

File: charts/tenant-base/chart/README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ The tenant base helm chart is a chart that give an easy all in one place to make
 
 ## Release Notes
 This section will describe the changes done in each helm chart release
+### 0.12.2
+- Sets the default PVC storageclassname to "". This enforces pvc's storageclasses to be default storageclass. This change does not affect tenants.
```
Comment (jaxels10) [inline charts/tenant-base/chart/README.md:7]:
```suggestion
- Sets the PVC storageclassname to "". This enforces pvc's storageclasses to be the default storageclass on the cluster. This change does not affect tenants, however, if they want to use a PVC they must upgrade to this version.
```

**Review by jaxels10 — COMMENTED** [review]
Just one small change before approving :)

**Review by jaxels10 — APPROVED** [review]
LGTM

**Pull request merged**